dot/parachain/availability-store/availability_store_test.go

```diff
@@ -1047,7 +1047,7 @@ func (h *testHarness) importLeaf(t *testing.T, parentHash common.Hash,
 	return aLeaf
 }
 
-func (h *testHarness) hasAllChunks(candidateHash parachaintypes.CandidateHash, nValidators uint32,
+func (h *testHarness) hasAllChunks(candidateHash parachaintypes.CandidateHash, nValidators uint32, //nolint
```

Comment:
why did we need nolint here?

Reply:
the linter complains 
``` 
`(*testHarness).hasAllChunks` - `nValidators` always receives `nValidators` (`10`) (unparam)
```
since nValidators is always 10, but it won't be in the real world, so this will change.